File: 260126/1부터의 합/sum-from-the-one.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc=new Scanner(System.in);

        int N=sc.nextInt();
        int temp=0;

        for(int i=1; i<101; i++){
            temp+=i;
            if(temp>=N){
                System.out.println(i);
                break;
            }
        }
    }
}

File: 260224/1부터의 합/sum-from-the-one.cpp
#include <iostream>
using namespace std;

int main() {
    // Please write your code here.
    int n;

    cin>>n;

    int sum=0;
    for(int i=1; i<101; i++){
        if(sum+i<n){
            sum+=i;
            break;
        }
    }
    cout<<sum;
    return 0;
}